app/Console/UpdateUniqueOrganizers.php

```diff
@@ -0,0 +1,179 @@
+        }
+
+        $reportFile = fopen('update_unique_organizers_report.txt', 'wb');
+        foreach ($messages as $message) {
+            fwrite($reportFile, $message. \PHP_EOL);
+        }
+        \fclose($reportFile);
+
+        return 0;
+    }
+
+    private function getAllOrganizerEvents(int $offset): array
+    {
+        return $this->connection->createQueryBuilder()
+            ->select('uuid, payload')
+            ->from('event_store')
+            ->where('type = "' . self::ORGANIZER_CREATED . '"')
+            ->orWhere('type = "' . self::ORGANIZER_WEBSITE_UPDATED . '"')
+            ->setFirstResult($offset)
+            ->setMaxResults(self::MAX_RESULTS)
+            ->execute()
+            ->fetchAll(PDO::FETCH_ASSOC);
+    }
+
+    private function getAllOrganizerEventsCount(): int
+    {
+        return $this->connection->createQueryBuilder()
+            ->select('uuid')
+            ->from('event_store')
+            ->where('type = "' . self::ORGANIZER_CREATED . '"')
+            ->orWhere('type = "' . self::ORGANIZER_WEBSITE_UPDATED . '"')
+            ->execute()
+            ->rowCount();
+    }
+
+    /**
+     * @throws DBALException
+     * @throws UniqueConstraintViolationException
+     */
+    private function updateOrganizer(Uuid $organizerUuid, Url $organizerUrl): void
+    {
+        // There are 3 possible states:
+        // 1. The organizer uuid is present with the correct url
+        // 2. The organizer uuid is not present
+        // 3. The organizer uuid is present with another value
+
+        $existingOrganizerUrl = $this->connection->createQueryBuilder()
+            ->select('unique_col')
+            ->from('organizer_unique_websites')
+            ->where('uuid_col = :uuid')
+            ->setParameter('uuid', $organizerUuid->toString())
+            ->execute()
+            ->fetchAll(PDO::FETCH_COLUMN);
+
+        if ($existingOrganizerUrl === (string) $organizerUrl) {
+            return;
+        }
+
+        if ($existingOrganizerUrl === null) {
+            $this->connection
+                ->insert(
+                    'organizer_unique_websites',
+                    [
+                        'uuid_col' => $organizerUuid->toString(),
+                        'unique_col' => (string) $organizerUrl,
+                    ]
+                );
+
+            return;
+        }
+
+        $this->connection
+            ->update(
+                'organizer_unique_websites',
+                [
+                    'uuid_col' => $organizerUuid->toString(),
+                    'unique_col' => (string) $organizerUrl,
+                ],
+                [
+                    'uuid_col' => $organizerUuid->toString(),
+                ]
+            );
+    }
+
+    private function getOrganizerUuid(array $organizerCreatedEvent): Uuid
+    {
+        return Uuid::fromString($organizerCreatedEvent['uuid']);
+    }
+
+    private function getOrganizerWebsite(array $organizerCreatedEvent): Url
```

Comment:
Nit: The parameter should probably be `$organizerEvent`? Since `WebsiteUpdated` is not "Organizer created". (Also in the `getOrganizerUuid()` method)